src/neo/Network/P2P/Peer.cs

```diff
@@ -32,7 +32,6 @@ private class WsConnected { public WebSocket Socket; public IPEndPoint Remote; p
         private IActorRef tcp_listener;
         private IWebHost ws_host;
         private ICancelable timer;
-        protected ActorSelection Connections => Context.ActorSelection("connection_*");
```

Comment:
Please don't remove this line.

Reply:
We don't quite understand why we need to keep this line?